### ISSUE TITLE
Allow enabling auto-accept when accepting connection invitation

### ISSUE THREAD

**Issue by jakubkoci** [hyperledger/aries-framework-javascript#583]
I found the following `TODO` in `ConnectionResponseHandler`:

```
// TODO: should we only send ping message in case of autoAcceptConnection or always?
// In AATH we have a separate step to send the ping. So for now we'll only do it
// if auto accept is enable
```

I would expect that the connection will be completed after I call `acceptInvitation`. Maybe, there is a reason why we have a separate step in AATH. But, what if we allow to update `autoAcceptConnections` via a config parameter in `acceptInvitation`? Or, maybe, we should set `autoAcceptConnections` automatically when user calls `acceptInvitation`? 🤷‍♂️

**Comment by jakubkoci** [comment]
To put more context... I'm trying to make the follwing test pass: [make a connection based on OOB invitation and process requests after the acceptation](https://github.com/hyperledger/aries-framework-javascript/pull/531/files#diff-261e45361be29b0d7c6678c64e2586f7c34cb9ab2e684e34727aa55fa06ed6c2R270), where I call `receiveInvitation` and then `acceptInvitation` expecting that after `acceptInvitation` is called, the connection will be finished and oob message request processed.

**Comment by TimoGlastra** [comment]
The idea was that you set `autoAccept` when calling `createConnection` or `acceptInvitation` but I think allowing for `autoAcceptConnection` in `acceptInvitation` makes sense. Automatically setting auto accept when calling `acceptInvitation` feels like a bigger change as it changes how AFJ works. We then reduce the connection protocol a 2-step exchange from the POV of a framework user, which we haven't done with any of the other protocols. This is exactly where the agent auto accept config can be used for, to make it feel like connections only needes create invitation / accept invitation

**Comment by jakubkoci** [comment]
Fixed by #589
